components/hero.module.css

```diff
@@ -1,121 +1,108 @@
+.container {
+	width: 100%;
+	height: 100%;
+	position: relative;
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	justify-content: center;
+	background: center/150% no-repeat url('../public/assets/background-wave.png');
+}
+
 .logo_wrap {
-  position: relative;
-  top: 20%;
-  text-align: center;
+	position: absolute;
+	top: 25%;
```

Comment:
Let's adjust `top` value on tablets and other bigger screens.
It looks a bit close to the title. **Not a high priority.**